arci-gamepad-gilrs/src/lib.rs

```diff
@@ -278,17 +278,95 @@ impl GilGamepad {
     pub fn new_from_config(config: GilGamepadConfig) -> Self {
         Self::new(config.device_id, config.map)
     }
+
+    /// This is another implementation for send command all the times
```

Comment:
Is it difficult to reuse the code of `new()`? I feel that you copy the code too much.

Reply:
I think so too and tried to reduce code on below commit.